code/modules/overmap/ships/computers/helm.dm

```diff
@@ -27,7 +27,7 @@
 /obj/machinery/computer/ship/helm/terminal
 	name = "helm control terminal"
 	icon = 'icons/obj/machinery/modular_terminal.dmi'
-	icon_screen = "helm"
+	icon_screen = "nav"
```

Comment:
why?

Reply:
i actually have no idea why this is in here